### ISSUE TITLE
[PkgEval] ImplicitEquations may have a testing issue on Julia 0.3 (2015-06-26)

### ISSUE THREAD

**Issue by IainNZ** [jverzani/ImplicitEquations.jl#1]
[PackageEvaluator.jl](https://github.com/IainNZ/PackageEvaluator.jl) is a script that runs nightly. It attempts to load all Julia packages and run their tests (if available) on both the stable version of Julia (0.3) and the nightly build of the unstable version (0.4). The results of this script are used to generate a [package listing](http://pkg.julialang.org/) enhanced with testing results.

#### On Julia 0.3
* On **2015-06-25** the testing status was `Tests pass.`
* On **2015-06-26** the testing status changed to `Tests fail.`

*This issue was filed because your testing status became worse. No additional issues will be filed if your package remains in this state, and no issue will be filed if it improves. If you'd like to opt-out of these status-change messages, reply to this message saying you'd like to and @IainNZ will add an exception. If you'd like to discuss [PackageEvaluator.jl](https://github.com/IainNZ/PackageEvaluator.jl) please file an issue at the repository. For example, your package may be untestable on the test machine due to a dependency - an exception can be added.*

*Test log*:
```
>>> 'Pkg.add("ImplicitEquations")' log
INFO: Cloning cache of ImplicitEquations from git://github.com/jverzani/ImplicitEquations.jl.git
INFO: Cloning cache of ValidatedNumerics from git://github.com/dpsanders/ValidatedNumerics.jl.git
INFO: Installing FactCheck v0.2.8
INFO: Installing ImplicitEquations v0.0.1
INFO: Installing Polynomials v0.0.3
INFO: Installing Requires v0.1.2
INFO: Installing ValidatedNumerics v0.1.0
INFO: Package database updated
INFO: METADATA is out-of-date — you may not have the latest version of ImplicitEquations
INFO: Use `Pkg.update()` to get the latest versions of your packages

>>> 'Pkg.test("ImplicitEquations")' log
Julia Version 0.3.10
Commit c8ceeef* (2015-06-24 13:54 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
INFO: Testing ImplicitEquations
Warning: Method definition floor(Interval{T<:Real},) in module ValidatedNumerics at /home/vagrant/.julia/v0.3/ValidatedNumerics/src/intervals/interval_arithmetic.jl:129 overwritten in module ImplicitEquations at /home/vagrant/.julia/v0.3/ImplicitEquations/src/intervals.jl:18.
Warning: Method definition ceil(Interval{T<:Real},) in module ValidatedNumerics at /home/vagrant/.julia/v0.3/ValidatedNumerics/src/intervals/interval_arithmetic.jl:130 overwritten in module ImplicitEquations at /home/vagrant/.julia/v0.3/ImplicitEquations/src/intervals.jl:19.
ERROR: `make_interval` has no method matching make_interval(
 in OInterval at no file
 in GRAPH at /home/vagrant/.julia/v0.3/ImplicitEquations/src/tupper.jl:52
 in include at ./boot.jl:245
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:285
 in _start at ./client.jl:354fatal: error thrown and no exception handler available.
Base.MethodError(f=make_interval, args=(Int64, ValidatedNumerics.Interval{Int64}(lo=0, hi=16)))
==========================[ ERROR: ImplicitEquations ]==========================

failed process: Process(`/home/vagrant/julia/bin/julia /home/vagrant/.julia/v0.3/ImplicitEquations/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
INFO: No packages to install, update or remove
ERROR: ImplicitEquations had test errors
 in error at error.jl:21
 in test at pkg/entry.jl:718
 in anonymous at pkg/dir.jl:28
 in cd at ./file.jl:20
 in cd at pkg/dir.jl:28
 in test at pkg.jl:67
 in process_options at ./client.jl:213
 in _start at ./client.jl:354


>>> End of log
```

**Comment by IainNZ** [comment]
ValidatedNumerics was just updated, so thats probably the cause

**Comment by jverzani** [comment]
Thanks! Wool get at this soon.

**Comment by jverzani** [comment]
closed with dff8dd24993f93de5f6b28c3da556e72006acff6
